wifi_passview_v_0.2.bat

```diff
@@ -0,0 +1,84 @@
+@echo off
+color b
+echo Administrative permissions required. Detecting permissions.
+ping localhost -n 2 >NUL
+cls
+echo Administrative permissions required. Detecting permissions..
+ping localhost -n 2 >NUL
+cls
+echo Administrative permissions required. Detecting permissions...
+ping localhost -n 2 >NUL
+net session >nul 2>&1
+if %errorLevel% == 0 (
+echo Administrator privileges found!
+echo Starting the program...
+ping localhost -n 2 >NUL
+) else (
+cls
+color c
+echo Please run this script with administrator privileges.
+pause
+exit
+)
+cls
+echo ============================================
+echo ** WiFi Passview v 0.2 (zsecurity.org)
+echo ** A batch program to show the saved WiFi Passwords!
+echo ** Created by Waren Gonzaga
+echo ** Facebook: @warengonzagaofficialpage
+echo ** Twitter: @waren_gonzaga
+echo ** Github: @warengonzaga
+echo ** Email: warengonzaga.dev@gmail.com
+echo ********************************************
+echo ** Modified by keethesh
+echo ** Email: keethesh15@gmail.com
+echo ** Github: @keethesh
```

Comment:
Rebranding is available you don't need to add credits to yourself here. You're already a contributor of this repo. The information in the script is for example only and a contact info mine if in case they have question about my project. You can rebrand the production version of the script that's why I put gulp.js so you can customize and rebrand the whole script without changing the internal code.